### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by angatiabenson** [abdelrahmanbonna/security_plus#11]
I'm experiencing a build failure when using the security_plus plugin version 2.0.1 in my Flutter project.

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':security_plus'.
> Could not open cp_proj generic class cache for build file 'C:\Users\benson\AppData\Local\Pub\Cache\hosted\pub.dev\security_plus-2.0.1\android\build.gradle' (C:\Users\benson\.gradle\caches\8.3\scripts\bul30twwdgr1tlw6rukm6qbdz).
   > Build file 'C:\Users\benson\AppData\Local\Pub\Cache\hosted\pub.dev\security_plus-2.0.1\android\build.gradle' should not contain a package statement.
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':security_plus' of type org.gradle.api.Project.
   > Could not get unknown property 'android' for project ':security_plus' of type org.gradle.api.Project.

* Try:
> Run with --stacktrace option to get the stack trace.
```

**Comment by abdelrahmanbonna** [comment]
@angatiabenson  could you give me more details about the environment you have on your machine?

**Comment by angatiabenson** [comment]
C:\Users\benson>flutter --version
Flutter 3.24.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 603104015d (6 days ago) • 2024-10-24 08:01:25 -0700
Engine • revision db49896cf2
Tools • Dart 3.5.4 • DevTools 2.37.3

**Comment by angatiabenson** [comment]
I was able to fix the issue by deleting the `package android` line from the `android/build.gradle` file in my Pub cache. Maybe you could look into why the line is causing this issue.

**Comment by abdelrahmanbonna** [comment]
@angatiabenson thanks I fixed it and uploaded a new version. fixed in version 2.0.2
